### ISSUE TITLE
Regex Issue in the Makefile

### ISSUE THREAD

**Issue by ddale27** [Macaulay2/M2#3300]
I noticed a small issue with the Makefile in the M2/M2 directory while compiling Macaulay2 recently. Under the `check-libtool` target on line 89, there is a regex `1s/[^.]*\.[^.]*\.\([1-9][0-9]*\).*/\1/` that doesn't seem to be working properly. I am running Arch Linux, and the most recent version of libtool outputs
```
libtool (GNU libtool) 2.5.0.1-38c1-dirty
Written by Gordon Matzigkeit, 1996

Copyright (C) 2024 Free Software Foundation, Inc.

```
when running `libtool --version`.
I'm assuming that this regex is meant to grab the baby version `0`, but running `libtool --version | sed -e '1s/[^.]*\.[^.]*\.\([1-9][0-9]*\).*/\1/' -e '2,$d'` gives, `libtool (GNU libtool) 2.1`. This causes issues with the `test` block on lines 90-92, causing the build checks to fail, even though my version of libtool is clearly recent enough to compile Macaulay2.
I think that the issue here is the `[1-9]` pattern in the parenthesis group, since the baby version is `0`, meaning it clearly wouldn't match `[1-9][0-9]*`. Possibly the pattern `[1-9]` should be removed from this regex and the one on line 88 since it is possible for a minor or baby version to be `0`, but I'm not confident enough with messing with regexes to try to fix it myself.

**Comment by d-torrance** [comment]
Thanks for the report!  This is fixed in #3302.

**Comment by d-torrance** [comment]
Closed by #3302.
